cypress/e2e/supervision/orders/bonds/edit-bond.cy.js

```diff
@@ -0,0 +1,33 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createClient()
+    .withOrder()
+    .withBond();
+});
+
+describe(
+  "Edit bond",
+  () => {
+    Cypress._.times(50, () => {
+      it("Triggering the validation summary when editing a bond in supervision", () => {
+        cy.get("@client").then(({id: clientId}) => {
+          cy.visit(
+            `/supervision/#/clients/${clientId}`
+          );
+        });
+
+        cy.get(".TABS_ORDERS").click();
+        cy.get("#tab-order-list-in-page-notification").should("contain", "Please wait...");
+        cy.get("#tab-order-list-in-page-notification").should("not.contain", "Please wait...");
+
```

Comment:
Is this just to wait for it to load? It's a little confusing we want it to contain but also not contain the words

Reply:
Aye it is indeed